android/sdl_android/src/main/java/com/smartdevicelink/encoder/VirtualDisplayEncoder.java

```diff
@@ -130,11 +130,6 @@ public void setStreamingParams(int displayDensity, ImageResolution resolution, i
         this.streamingParams = new VideoStreamingParameters(displayDensity, frameRate, bitrate, interval, resolution, format);
     }
 
-    @SuppressWarnings("unused")
-    public void setStreamingParams(int displayDensity, ImageResolution resolution, int frameRate, int bitrate, int interval, VideoStreamingFormat format, boolean stableFrameRate) {
-        this.streamingParams = new VideoStreamingParameters(displayDensity, frameRate, bitrate, interval, resolution, format, stableFrameRate);
-    }
-
```

Comment:
Hello @shiniwat, even though this method is currently unused I believe it should be kept because it will replace the deprecated method above it when that method is eventually removed.

Reply:
Understood. I have restored setStreamingPrams back accordingly.